src/components/view-switcher/ViewSwitcher.jsx

```diff
@@ -0,0 +1,37 @@
+import ToggleButton from '@mui/material/ToggleButton'
+import Box from '@mui/material/Box'
+
+import FormatListBulletedIcon from '@mui/icons-material/FormatListBulleted'
+import GridViewIcon from '@mui/icons-material/GridView'
+
+import { cardsViews } from '~/constants'
+import { styles } from '~/components/view-switcher/ViewSwitcher.styles'
+
+const ViewSwitcher = ({ setOffersView, offersView }) => {
+  const changeOffersView = (_event, view) => setOffersView(view)
```

Comment:
It should emit to parent and not set state...

Reply:
you mean it has to be incoming function that arguments will take not only one view but also spreading incoming other filters or what?  @maarekua